Packs/DigitalShadows/pack_metadata.json

```diff
@@ -34,11 +34,13 @@
     ],
     "githubUser": [
         "ds-jdcockrill",
-        "rossbowerman"
+        "rossbowerman",
+        "rq-vsarode"
     ],
     "devEmail": [
         "jamie.cockrill@digitalshadows.com",
```

Comment:
Maybe we should delete irrelevant contacts here, if any?

Reply:
yes make sense